project_script/datatreatment.py

```diff
@@ -19,3 +21,35 @@ def is_bigger_data_frame(data_frame_1, data_frame_2):
 
 def equalize_data_frame_rows(data_frame_1, data_frame_2):
     return data_frame_1.loc[0:len(data_frame_2)-1, :]
+
+
+def smooth(array):
```

Comment:
What kind of data this function smooth? Maybe the name could be more readable. Ex: `smooth_something`, `smooth_columns`.

What the parameter array means? If you change the name to the real name that the array carry, could be more readable. Ex: cars, names, data_frame, hours